src/gas/gas.cpp

```diff
@@ -136,10 +154,10 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   std::string scattering_model_name =
       pin->GetOrAddString("gas/opacity/scattering", "scattering_model", "none");
   if (scattering_model_name == "none") {
-    scattering = singularity::photons::GrayS(0.0, 1.0);
+    scattering = NonCGSUnitsS<GrayS>(GrayS(0.0, 1.0), time, mass, length, 1.);
   } else if (scattering_model_name == "constant") {
     const Real kappa_s = pin->GetOrAddReal("gas/opacity/scattering", "kappa_s", 0.0);
-    scattering = singularity::photons::GrayS(kappa_s, 1.0);
+    scattering = NonCGSUnitsS<GrayS>(GrayS(kappa_s, 1.0), time, mass, length, 1.);
```

Comment:
Just to be clear, even if artemis is using cgs units, this is still called `NonCGSUnits` right?

Reply:
Yes, this might be better called `CustomUnits` on the singularity side. I made the choice to always include the unit conversions to halve the number of singularity variants we're carrying around.